submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+
+However that definition is still a bit vague: As someone who didn't play it, would Dwarf Fortress' gameplay of enabling dwarfs to *make stuff* would fall under my definition of automation? But let's not get to far from Factorio here, maybe someone else will explore this further in the future. *\*wink wink\**
+
+### There's a mod for it
+
+I've already touch on Minecraft as an example for automation, and it's popularity boom in 2013 and with it one of the largest modding communities ever[^2][^3] could just have made it the most popular example of automation. But that got me thinking: With mods in the likes of Buildcraft, IndustrialCraft, and Redpower featuring a very heavy focus on industrial machinery and automation, and release dates well before Factorio's development even started,[^4] could that have been an inspiration for Factorio?
+
+So I went on a search (which definitely wasn't easy, seeing how active some of Wube's devs are across Reddit, Discord and the forum), and after sifting through a few interviews and clicking on google page 2, I finally found something: An [old forum thread](https://forums.factorio.com/viewtopic.php?f=5&t=3026) from March 2014, confirming Buildcraft (or IndustrialCraft, according to a different interview) as the main inspiration to Factorio.[^5][^6] According to kovarex, he saw the potential in that concept while playing the mod, and wanted more of it (in his own spin, of course).
+
+While I had hoped to find an answer, I for sure wasn't expecting to find such a direct one. Having that out of the way, though, we can definitely speculate a bit more on brought Factorio to were it's at today.
+
+### Great games design themselves
+
+There is a principle in game design designers should follow, that goes along the lines "follow the fun",[^7] meaning you shouldn't try to force a game to be something it isn't, but rather develop on the ideas that are enjoyable. While examples for that statement often include dramatic changes like Ape Out went through,[^8] it still applies to games that don't shift that drastically in direction. Let's take Factorio itself as an example:  Originally the game had multiple pickaxes – a wooden, which you could then craft into a better one out of stone, and then into the iron pickaxe we had for such a long time [^citation needed]. This was clearly inspired by Minecraft (and also confirmed by the devs[^9]), but having so many items about manual mining in a game about automation gives the idea about the game – it draws the fun away. So they simplified it, gave you just your iron and steel pickaxe, but that's still two items specifically about manual mining. Thus they removed pickaxes completely from the game in `0.17.0`[^10], and with all focus on the unfun, manual labor.
```

Comment:
You could add a little thing here about Factorio's philosophy. They're definitely following the fun, but taking great care not to give in too much. They talk a lot about how gamers will always "optimize the fun out of everything", and I'm very sure I've seen kovarex and even bilka mention this somewhere. But I really can't remember for what aspect of the game.